SPECv2.md

```diff
@@ -70,9 +70,9 @@ After these a number of fields will exist.
 | Preview secs  | 1        | 'P'   | uint8   | Number of seconds of recording before motion event was detected
 | Latitude      | 4        | 'L'   | float32 | Latitude of device location
 | Longitude     | 4        | 'O'   | float32 | Longitude of device location
-| LocTimestamp  | 8        | 'S'   | uint64  | Time at which location of device was set.  Microseconds since 1970-01-01 UTC
+| LocTimestamp  | 8        | 'S'   | uint64  | Time at which location of device was set.
```

Comment:
Why did you remove the microseconds bit? It's useful to explain the units.

Reply:
Well I use the golang time.Now() function to set the timestamp.

And I'm not sure if that is microseconds since 1970-01-01 UTC.

I'll look into this a bit further today.

Reply:
If it's being encoded in the same way as recording timestamp then it'll be microseconds since 1970-01-01 in the CPTV file.

Reply:
Ok, will check it.

Reply:
You were right.